packages/website-eslint/package.json

```diff
@@ -31,13 +31,15 @@
     "@typescript-eslint/eslint-plugin": "6.2.0",
     "@typescript-eslint/parser": "6.2.0",
     "@typescript-eslint/scope-manager": "6.2.0",
-    "@typescript-eslint/types": "6.0.0",
```

Comment:
These packages had been duplicated with the `dependencies` and were not being updated, causing issues. Just needed to remove them